### DISCUSSION TITLE
Is e2ee here yet?

### DISCUSSION THREAD

**Discussion by AtFreezingPoint** [stoatchat/discussions#597] [Q&A] [answered]
Last i used revolt was like 6 months ago, is e2ee for private conversations here yet?

**Comment by Rexogamer** [comment, marked as answer]
Not yet - we're currently working on the new client (you can find the public progress [here](https://github.com/revoltchat/frontend)) and the voice server rewrite. We hope to add E2EE at some point in the future, but we do not have any ETAs.

**Reply by Yolakalemowa** [reply]
any updates on this? 2 years later?

**Reply by insertish** [reply]
As our roadmap stands right now, we expect to begin work on e2ee after we release video chats.

**Reply by Yolakalemowa** [reply]
> As our roadmap stands right now, we expect to begin work on e2ee after we release video chats.

hi thank you so much for answering. I get confused at how to follow up on github esp. after revolt rebranding. Could you tell me if there's even a very general ETA of video chats and e2ee? And location of roadmap that I can follow? Really appreciate it.

**Reply by DeclanChidlow** [reply]
Still no ETAs. Stoat is supported by donations and developed by volunteers, so any times that could be provided would be based on little certainty.
